src/PostgREST/App.hs

```diff
@@ -41,8 +40,7 @@ import qualified PostgREST.Response   as Response
 import qualified PostgREST.Unix       as Unix (installSignalHandlers)
 
 import PostgREST.ApiRequest           (ApiRequest (..))
-import PostgREST.AppState             (AppState)
-import PostgREST.Auth                 (AuthResult (..))
+import PostgREST.AppState             (AppState, AuthResult (..))
```

Comment:
How about removing these new indentations for another commit? Maybe codecov is complaining because it's detecting these files? Not sure, but also it's better to make these as "refactor: indent imports".

Reply:
> Which is... why are we exporting it from both places? That doesn't make sense to me.

I did this a while ago in #2928. I think I naively did this to avoid a circular dependency problem.

Reply:
@taimoorzaeem Cool. Btw, if you can make refactors in separate PRs please go ahead, reviews are easier this way too.